radix-engine-tests/tests/system/crypto_utils.rs

```diff
@@ -90,15 +156,118 @@ fn test_crypto_scrypto_verify_bls12381_v1() {
     assert!(!msg2_verify);
 }
 
+#[test]
+fn test_crypto_scrypto_bls12381_aggregate_verify() {
```

Comment:
Minor: I think it would be better to structure this test as `Arrange`, `Act`, `Assert`, `Act`, `Assert`

Reply:
good point, will rework it